projects/storefrontapp-e2e-cypress/cypress/integration/mobile/my-account/update-password-mobile.e2e-spec.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  updatePasswordTest,
+  verifyAsAnonymous,
+} from '../../../helpers/update-password';
+import * as login from '../../../helpers/login';
+import * as homepage from '../../../helpers/homepage';
+import { formats } from '../../../sample-data/viewports';
+
+describe(`${formats.mobile.width +
+  1}p resolution - Update Password page`, () => {
+  before(() => {
+    cy.window().then(win => win.sessionStorage.clear());
+    cy.viewport(formats.mobile.width, formats.mobile.height);
+  });
+
+  verifyAsAnonymous();
+
+  describe('update password test for logged in user', () => {
+    before(() => {
+      cy.viewport(formats.mobile.width, formats.mobile.height);
+      homepage.clickHamburger();
+    });
+
+    beforeEach(() => {
+      cy.restoreLocalStorage();
+      cy.viewport(formats.mobile.width, formats.mobile.height);
```

Comment:
we should move this above to the global one in a beforeEach and keep this consistency